lib/fastlane_core/pkg_file_analyser.rb

```diff
@@ -0,0 +1,39 @@
+require 'nokogiri'
+
+module FastlaneCore
+  class PkgFileAnalyser
+    def self.fetch_app_identifier(path)
+      xml = self.fetch_distribution_xml_file(path)
+      return xml.at_css('product')['id'] if xml
+      return nil
+    end
+
+    # Fetches the app version from the given pkg file.
+    def self.fetch_app_version(path)
+      xml = self.fetch_distribution_xml_file(path)
+      return xml.at_css('product')['version'] if xml
```

Comment:
If `produce` is nil, this will crash, maybe we should add some error handling for that?